templates/python_binary.cpp.in

```diff
@@ -4,25 +4,26 @@
 /// @author Simon Heybrock
 #include "pybind11.h"
 
-#include "scipp/dataset/@NAME@.h"
-#include "scipp/variable/@NAME@.h"
+#include "scipp/dataset/@OPNAME@.h"
+#include "scipp/variable/@OPNAME@.h"
 
 using namespace scipp;
 
 namespace py = pybind11;
 
-template <typename T> void bind_@NAME@(py::module &m) {
+template <class T1, class T2 = T1> void bind_@OPNAME@(py::module &m) {
   m.def(
-      "@NAME@", [](
-        const typename T::const_view_type &a,
-        const typename T::const_view_type &b) {
+      "@OPNAME@", [](
+        const typename T1::const_view_type &a,
+        const typename T2::const_view_type &b) {
       return @NAME@(a,b);
       },
       py::arg("a"),py::arg("b"), py::call_guard<py::gil_scoped_release>());
```

Comment:
add `py::is_operator`? Or is this template also used for non-operator functions?

Actually, how is the name handled? `@OPNAME@` expands to e.g. `plus`. But the function name needs underscores. What am I missing?

Reply:
This generates/bins the named functions, the bindings for the operators such as `__add__` are unfortunately not generated yet, but created by hand (using templates from `bind_operators.h`).